lib/src/blockdev.rs

```diff
@@ -37,13 +37,14 @@ pub(crate) struct Device {
     // Filesystem-related properties
     pub(crate) label: Option<String>,
     pub(crate) fstype: Option<String>,
+    pub(crate) path: Option<String>,
 }
 
 impl Device {
     #[allow(dead_code)]
     // RHEL8's lsblk doesn't have PATH, so we do it
```

Comment:
We could probably not care about RHEL8 anymore, and drop the `Option` from the `path` too. But fine as is!